### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by tienv9**
Website: https://jolly-sand-01311d41e.3.azurestaticapps.net/

Woodle game ideas : 
-A hint system to give a letter hint or a correct position if the player makes no progress in 2-3 tries
-A chaos mode where the word is spell backward 
-A message at the end of the game depending on the player's completion of the word. ex: "Congrats", "You're almost there", or "You're not even trying are you?" for 0 correct letters.

I have no idea if this is correct or not and totally forgot about the PR turn in date because I look at canvas for the due date

### CODE REVIEW

**Comment by CKappes-Git** [comment]
# EWU-CSCD378-2023-Spring

## Assignment 1

The purpose of this assignment is to solidify your learning of:

- Building a Vue 3, Vuetify, TypeScript App
- Unit testing infrastructure using Jest
- Creating a CI/CD pipeline with GitHub Actions
- Deploying the site to Azure

## Instructions

1. Create a Vue app ✅
   - Uses TypeScript ✅
   - Add Vuetify ✅
   - Uses Jest ❌

3. Create a simple page called game.vue ✅
   - Page should have a card with a heading that with a clever name for the game ✅
   - some default placeholder body text ✅
   - and a button to take you back to the home page ✅
   - there should be a button on the home page to access this page ✅

4. Check all the code into your repo fork ✅

5. Create a static web app in Azure ✅
   - Create the container where your GitHub repo will deploy
   - Link the site with your GitHub repo

6. Update the CI/CD pipeline ✅
   - Pipeline runs on each check in
   - Jest tests run and pass ❌
   - Vue app builds ✅
   - Vue app deploys to Azure ✅

7. Azure site works ✅
   - Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

## Extra Credit

- Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
- One point per idea up to three points

The app looks good to me and I'm assuming the pipeline is working (although I'll have to ask the teacher how to check that).  Although, and I might have just missed it, I didn't see any test files for using Jest.

**Comment by tienv9** [comment]
> # EWU-CSCD378-2023-Spring
> ## Assignment 1
> The purpose of this assignment is to solidify your learning of:
> 
> * Building a Vue 3, Vuetify, TypeScript App
> * Unit testing infrastructure using Jest
> * Creating a CI/CD pipeline with GitHub Actions
> * Deploying the site to Azure
> 
> ## Instructions
> 1. Create a Vue app ✅
>    
>    * Uses TypeScript ✅
>    * Add Vuetify ✅
>    * Uses Jest ❌
> 2. Create a simple page called game.vue ✅
>    
>    * Page should have a card with a heading that with a clever name for the game ✅
>    * some default placeholder body text ✅
>    * and a button to take you back to the home page ✅
>    * there should be a button on the home page to access this page ✅
> 3. Check all the code into your repo fork ✅
> 4. Create a static web app in Azure ✅
>    
>    * Create the container where your GitHub repo will deploy
>    * Link the site with your GitHub repo
> 5. Update the CI/CD pipeline ✅
>    
>    * Pipeline runs on each check in
>    * Jest tests run and pass ❌
>    * Vue app builds ✅
>    * Vue app deploys to Azure ✅
> 6. Azure site works ✅
>    
>    * Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
> 
> ## Extra Credit
> * Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
> * One point per idea up to three points
> 
> The app looks good to me and I'm assuming the pipeline is working (although I'll have to ask the teacher how to check that). Although, and I might have just missed it, I didn't see any test files for using Jest.

@CKappes-Git This is an old Assigment1.md, and the assignment now requires using Vitest instead of Jest.

File: wordle-web/src/views/Game.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  
+        <v-card-actions>
+          <v-spacer />
+          <RouterLink to="/">Home<v-btn text color="primary"></v-btn> </RouterLink>
+        </v-card-actions>
+      </v-card>
+    </div>
+  </template>
+  
+  <style>
```
Comment (mmwoodfo) [inline wordle-web/src/views/Game.vue:19]: I don't think this is doing anything

File: wordle-web/src/views/Game.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <v-card-title> Welcome to bear royale! </v-card-title>
+        <v-card-subtitle> (game idea) </v-card-subtitle>
+  
+        <v-card-text>
+          Be the last bear in your area in an arena of 3 bears. 
+        </v-card-text>
+  
+        <v-card-actions>
+          <v-spacer />
+          <RouterLink to="/">Home<v-btn text color="primary"></v-btn> </RouterLink>
```
Comment (mmwoodfo) [inline wordle-web/src/views/Game.vue:13]:
```suggestion
          <v-btn text color="primary"  to="/">Home</v-btn>
```

File: wordle-web/src/views/Game.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="game">
```
Comment (mmwoodfo) [inline wordle-web/src/views/Game.vue:2]: You don't need this div

**Comment by mmwoodfo** [comment]
You should be making your PR against the assignment 1 branch

**Pull request closed without merging**